pygeoapi/templates/collections/items/index.html

```diff
@@ -140,6 +146,32 @@ <h1>{% for l in data['links'] if l.rel == 'collection' %} {{ l['title'] }} {% en
     </div>
     {% endif %}
     </section>
+    <script>
+    	$(document).ready(function() {
+      	// Setup - add a text input to each footer cell
+      	$('#searchTable thead th').each(function(i) {
+        	var title = $('#searchTable thead th').eq($(this).index()).text();
+        	$(this).html('<input type="text" placeholder="Search ' + title + '" data-index="' + i + '" />');
+      	});
+
+      	// DataTable
+      	var table = $('#searchTable').DataTable({
+        	responsive: true,
+        	paging: true,
+        	sorting: false,
```

Comment:
Will setting this to true enable sorting? If so, that's an easy win.

Reply:
There's an issue where when you click in the search bar to type what you want to find it also sorts by that column which I am unsure how to address right now.